### PR TITLE
lib: pelion: Disable mbedtls prompt off

### PR DESCRIPTION
**Pull request by pdunaj**
Disable prompt off until the option is available.

Signed-off-by: Pawel Dunaj <pawel.dunaj@nordicsemi.no>

### CODE REVIEW

**Review by tejlmand — APPROVED** [review]

File: lib/pelion/Kconfig [hunk 1]
```diff
@@ -89,7 +89,7 @@ config PELION_NRF_SECURITY
 config PELION_MBEDTLS
 	bool "mbedTLS built-in"
 	select OPENTHREAD_MBEDTLS if NET_L2_OPENTHREAD
-	select MBEDTLS_CFG_FILE_PROMPT_OFF
+#	select MBEDTLS_CFG_FILE_PROMPT_OFF
```
Comment (tejlmand) [inline lib/pelion/Kconfig:92]: actually, this line should be deleted completed.
The `MBEDTLS_CFG_FILE_PROMPT_OFF` was removed during rework of initial approach.

```suggestion
```

**Review by tejlmand — CHANGES_REQUESTED** [review]

File: lib/pelion/Kconfig (hunk 1 above)
Comment (pdunaj) [reply lib/pelion/Kconfig]: done

**Review by carlescufi — APPROVED** [review]

**Review by tejlmand — APPROVED** [review]

**Comment by tejlmand** [comment]
maybe commit message is a little off now 😉 
>  lib: pelion: Disable mbedtls prompt off
> 
> Disable prompt off until the option is available.

**Comment by pdunaj** [comment]
> maybe commit message is a little off now 😉
> 
> > lib: pelion: Disable mbedtls prompt off
> > Disable prompt off until the option is available.

True but let's leave it. We are not exactly lying there.

**Pull request merged**